precompiles/bank/bank.go

```diff
@@ -75,6 +75,10 @@ func (Precompile) Address() common.Address {
 
 // RequiredGas calculates the precompiled contract's base gas rate.
 func (p Precompile) RequiredGas(input []byte) uint64 {
+	if len(input) < 4 {
+		return 0
+	}
+
```

Comment:
without adding this logic in the precompiles trying to send a transfer to this address would cause a panic instead of returning an error gracefully

Reply:
can you please add this as a comment in the code?